tests/pytests/test_issues.py

```diff
@@ -975,3 +975,30 @@ def test_mod5910(env):
     env.assertEqual(iterators_profile[1][1], 'INTERSECT')
     env.assertEqual(iterators_profile[1][7][1], 'NUMERIC')
     env.assertEqual(iterators_profile[1][8][1], 'UNION')
+
+
+@skip(cluster)
```

Comment:
what does that do?
Did you mean `@skip(cluster=True)` or `@skip(cluster=False)`?

Reply:
Currently to skip also when False, we need to add something with 3-state such as https://github.com/RediSearch/RediSearch/pull/4166/files#diff-48a80e3d857e932ed92f9342791cea987617fa88ca398d0df7b7efffb7972b6fR300

Reply:
Ahead of you. On master (soon if not already on 2.6, 2.8) we have it :sunglasses: